### PR TITLE
[RSDK-1055] update SLAM integration tests

### PR DESCRIPTION
**Pull request by JohnN193**
updates integration tests with new endpoints to prep for map representation wrap up. Does not remove old endpoints, as those should be removed in [RSDK-1066](https://viam.atlassian.net/browse/RSDK-1066?atlOrigin=eyJpIjoiMjgwNjQ4MWM0NmUzNGM1OThkZWMzN2JlYTJhMjZhZjUiLCJwIjoiaiJ9). In addition, the internal state and pointcloud endpoints were [added previously](https://viam.atlassian.net/browse/RSDK-1955?atlOrigin=eyJpIjoiOWYyZDY1NTFiOTBjNDkwOGJkOGI0ODJkMjIxOTRjZmYiLCJwIjoiaiJ9), so this PR only adds the GetPosition method

tickets:
[cartographer integration test](https://viam.atlassian.net/browse/RSDK-1055)
[orbslam integration test](https://viam.atlassian.net/browse/RSDK-1067?atlOrigin=eyJpIjoiMjg2OGZlNWRiOGZkNGQ4MzllY2Q4YjdmNzA2MTJlYWUiLCJwIjoiaiJ9)

[RSDK-1066]: https://viam.atlassian.net/browse/RSDK-1066?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: services/slam/builtin/orbslam_int_test.go [hunk 1]
```diff
@@ -89,7 +89,7 @@ func testOrbslamMap(t *testing.T, svc slam.Service) {
 }
 
 // Checks the orbslam position within a defined tolerance
-func testOrbslamPosition(t *testing.T, svc slam.Service, mode, actionMode string) {
+func testOrbslamPosition(t *testing.T, svc slam.Service, mode, actionMode string, sensor string) {
```
Comment (nicksanford) [inline services/slam/builtin/orbslam_int_test.go:92]: Where is `sensor` used?

File: services/slam/builtin/cartographer_int_test.go [hunk 2]
```diff
@@ -45,22 +45,44 @@ func testCartographerMap(t *testing.T, svc slam.Service) {
 }
 
 // Checks the cartographer position within a defined tolerance.
-func testCartographerPosition(t *testing.T, svc slam.Service) {
-	expectedPos := r3.Vector{X: -0.004, Y: 0.004, Z: 0}
+func testCartographerPosition(t *testing.T, svc slam.Service, sensor string) {
```
Comment (nicksanford) [inline services/slam/builtin/cartographer_int_test.go:48]: Where is sensor used?

**Comment by nicksanford** [comment]
@JohnN193 I think this branch is the same branch you used here:  https://github.com/viamrobotics/rdk/pull/1965

Mind using a different branch?

**Review by nicksanford — CHANGES_REQUESTED** [review]

File: services/slam/builtin/orbslam_int_test.go (hunk 1 above)
Comment (JohnN193) [reply services/slam/builtin/orbslam_int_test.go:92]: Ah, I was supposed to test `componentRef` against sensor. Thanks for catching this!

**Comment by JohnN193** [comment]
Closing this to switch to a new branch

**Pull request closed without merging**